src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCode.cs

```diff
@@ -159,11 +150,12 @@ internal static bool OpcodeBacktracks(int Op)
             }
         }
 
-        internal static int OpcodeSize(int Opcode)
+#if DEBUG
+        internal static int OpcodeSize(int opcode)
```

Comment:
`OpcodeSize` is used by `RegexCompiler` in the desktop code... Here, it's only used in debug builds. Not sure if it should stay in release builds here or if it can be changed to debug-only.